scripts/basic-workflow-tests.sh

```diff
@@ -213,19 +212,34 @@ test_exec() {
     kill $PID
 }
 
+wait_for_server(){
+    if ! timeout 120 bash -c '
+        until curl -sS http://localhost:8000/docs &> /dev/null; do
```

Comment:
Do models served with vLLM have a docs page? Setting the address being checked to `http://localhost:8000/v1/models`  mirrors the checks we have internally in `list_models`.

Reply:
Yes, initially I thought it wasn't there but I tested it and `/docs` is present.